cmd/root.go

```diff
@@ -81,7 +81,7 @@ func doIt(cmd *cobra.Command, args []string) {
 		bomb(err.Error())
 	}
 
-	if errCount > 0 {
+	if errCount > 0 && *flags.ErrorStatusOnFail {
```

Comment:
Deferencing a ptr without checking! Granted this might be ok given it will be init by cobra but let's just check for good measure.